### DISCUSSION TITLE
Url's with and without trailing /

### DISCUSSION THREAD

**Discussion by kasperaamodt** [plausible/analytics#1399] [Self-Hosted Support] [answered]
Hi,

Just noticed that urls with a trailing /, and urls without a trailing /, are counted as different pages. Is ut possible to fix this?

![Screenshot 2021-10-20 at 09 48 21](https://user-images.githubusercontent.com/44305520/138051019-b0518d97-941d-4c13-a715-499953ec1d39.png)

**Comment by metmarkosaric** [comment, marked as answer]
thanks @kasperaamodt! this is a technical issue on your site and it's best to be fixed directly on your site to avoid other conflicts such as problems with SEO. we do plan to look at it if we can hide the technical problem for you on our dashboard in the future but always best to actually fix it. there's a thread on this already #336
